pkgs/development/tools/misc/kibana/6.x.nix

```diff
@@ -1,17 +1,45 @@
-{ stdenv, makeWrapper, fetchurl, elk6Version, nodejs, coreutils, which }:
+{ elk6Version
+, enableUnfree ? true
```

Comment:
Why would we want `pkgs.kibana` to be unfree by default? We don't enable allowUnfree by default so this package wouldn't be available.

Reply:
Also see: https://github.com/NixOS/nixpkgs/commit/3f889f30bee92c3f2d5eaf53e56aef68f61efc4a